e2e/Makefile

```diff
@@ -7,10 +7,13 @@ cleanup-ibc-test-containers:
 		$(DOCKER) rm $$id ; \
 	done
 
-e2e-test: cleanup-ibc-test-containers
+init:
+	./scripts/init.sh
+
+e2e-test: init cleanup-ibc-test-containers
```

Comment:
addition of `init` here means we shouldn't need to do anything to get the default config created.